src/frontend/src/pages/FinancePage/CreateReimbursementRequestForm/CreateReimbursementFormView.tsx

```diff
@@ -0,0 +1,230 @@
+import { Delete } from '@mui/icons-material';
+import {
+  FormControl,
+  FormHelperText,
+  FormLabel,
+  Grid,
+  IconButton,
+  MenuItem,
+  Select,
+  TextField,
+  Typography
+} from '@mui/material';
+import { Box, Stack } from '@mui/system';
+import { Control, Controller, FieldErrors, UseFormHandleSubmit } from 'react-hook-form';
+import { ClubAccount, ExpenseType, ReimbursementProductCreateArgs, Vendor, WbsNumber, wbsPipe } from 'shared';
+import { DatePicker } from '@mui/x-date-pickers';
+import ReimbursementProductTable from './ReimbursementProductTable';
+import NERFailButton from '../../../components/NERFailButton';
+import NERSuccessButton from '../../../components/NERSuccessButton';
+import { routes } from '../../../utils/routes';
+import { CreateReimbursementRequestFormInput } from './CreateReimbursementRequestForm';
+import PageLayout from '../../../components/PageLayout';
+
+interface CreateReimbursementRequestFormViewProps {
+  allVendors: Vendor[];
+  allExpenseTypes: ExpenseType[];
+  receiptFiles: {
+    file: File;
+  }[];
+  allWbsElements: {
+    wbsNum: WbsNumber;
+    wbsName: string;
+  }[];
+  control: Control<CreateReimbursementRequestFormInput, any>;
+  reimbursementProducts: ReimbursementProductCreateArgs[];
+  receiptAppend: (args: { file: File }) => void;
+  receiptRemove: (index: number) => void;
+  reimbursementProductAppend: (args: ReimbursementProductCreateArgs) => void;
+  reimbursementProductRemove: (index: number) => void;
+  onSubmit: (data: CreateReimbursementRequestFormInput) => void;
+  handleSubmit: UseFormHandleSubmit<CreateReimbursementRequestFormInput>;
+  errors: FieldErrors<CreateReimbursementRequestFormInput>;
+  totalCost: number;
+}
+
+const CreateReimbursementRequestFormView: React.FC<CreateReimbursementRequestFormViewProps> = ({
+  allVendors,
+  allExpenseTypes,
+  allWbsElements,
+  receiptFiles,
+  reimbursementProducts,
+  control,
+  receiptAppend,
+  receiptRemove,
+  reimbursementProductAppend,
+  reimbursementProductRemove,
+  onSubmit,
+  handleSubmit,
+  errors,
+  totalCost
+}) => {
+  const wbsElementAutocompleteOptions = allWbsElements.map((wbsElement) => ({
+    label: wbsPipe(wbsElement.wbsNum) + ' - ' + wbsElement.wbsName,
+    id: wbsPipe(wbsElement.wbsNum)
+  }));
+
+  const ReceiptFileInput = () => (
+    <>
+      <FormLabel>Receipts</FormLabel>
+      <ul>
+        {receiptFiles.map((receiptFile, index) => (
+          <li>
```

Comment:
each list item should have a unique key

Reply:
I like how it looks now, I know u mentioned adding the preview on here, that's up to u, I think it's fine with our without for now, but we'd need to adjust the design a little to accomodate that